lib/mockbuilder.py

```diff
@@ -57,7 +57,6 @@ def initialize(self):
 
     def build(self, package):
         LOG.info("%s: Starting build process" % package.name)
-        self._prepare(package)
```

Comment:
That was a mistake indeed. The PR is already merged though, so there's nothing I can do.